src/main/java/com/palehorsestudios/alone/player/Player.java

```diff
@@ -157,6 +158,11 @@ public boolean isDead() {
     } else if (this.hydration <= 0) {
       gameOver = true;
     }
+    if (gameOver) {
```

Comment:
I think we should move changing visibility of the fields to GameApp. This would improve our separation of concerns.